chainercv/visualizations/vis_bbox.py

```diff
@@ -1,25 +1,44 @@
+import numpy as np
+
 from chainercv.visualizations.vis_image import vis_image
 
 
-def vis_bbox(img, bbox, label=None, score=None, label_names=None, ax=None):
+def vis_bbox(img, bbox, label=None, score=None, label_names=None,
+             instance_colors=None, alpha=1., linewidth=3., ax=None):
     """Visualize bounding boxes inside image.
 
     Example:
 
-        >>> from chainercv.datasets import VOCDetectionDataset
+        >>> from chainercv.datasets import VOCBboxDataset
         >>> from chainercv.datasets import voc_bbox_label_names
         >>> from chainercv.visualizations import vis_bbox
         >>> import matplotlib.pyplot as plot
```

Comment:
`plot` -> `plt` (https://github.com/chainer/chainer/pull/3698)

Reply:
This has not been fixed yet in ChainerCV.